src/opm/parser/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -1211,6 +1215,48 @@ bool FieldProps::tran_active(const std::string& keyword) const {
     return calculator != this->tran.end() && calculator->second.size() > 0;
 }
 
+void FieldProps::applyNumericalAquifers(const NumericalAquifers& numerical_aquifers) {
+    // TODO: ideally, we should also update the cell_depth, but it is not used for later
+    // in the simulator. So we do not update it here.
+    // Maybe also cell volume? Not sure whether it is used later
+    auto& porv_data = this->init_get<double>("PORV").data;
+    auto& satnum_data = this->int_data["SATNUM"].data;
+    auto& pvtnum_data = this->int_data["PVTNUM"].data;
+    numerical_aquifers.updateCellProps(*(this->grid_ptr), porv_data, satnum_data, pvtnum_data);
+
+    this->updateTransWithNumericalAquifer(numerical_aquifers);
+}
+
+void FieldProps::updateTransWithNumericalAquifer(const NumericalAquifers& numerical_aquifers) {
+    const std::array<std::set<size_t>, 3> trans_to_remove = numerical_aquifers.transToRemove(*(this->grid_ptr));
```

Comment:
This can be `const T&`

Reply:
I will do it. But a question here, I understand with `const T&` will guarantee good performance here.  But with `const T` I thought `std::move` will be triggered.  Am I right about this?

Reply:
I am no moving export - but you can at least not move out of a `const` object.

Reply:
There is no `const` object. It was a temporary variable generated on-fly in the function `transToRermove`.  If I understand your comment correctly.